src/greenpak/i2c.py

```diff
@@ -48,11 +45,91 @@ def read(
         """
         assert False, f"Class {self.__class__} does not implement read()"
 
+    def write_read(self, addr: int, wreq: bytearray, rbytes: int,
```

Comment:
read_write is not part of the new interface which will include just the three methods gp_read, gp_write, and gp_scan. Can you modify this file to use only these three methods?